HISTORY.rst

```diff
@@ -3,6 +3,11 @@
 Release History
 ===============
 
+0.5.2 (2019-07-24)
+++++++++++++++++++
+
+- sapmonitor create will now accept key vault as well
```

Comment:
This sentence doesn't make sense on its own. I would suggest being more clear (e.g. accept key vault url for credentials)